locales/es/common.json

```diff
@@ -37,6 +37,7 @@
   "my-turn": "Mi Turno",
   "nearby-shops": "Comercios Cercanos",
   "next-turn": "Próximo Turno",
+  "no-shops": "Parece que no todavía no hay ningún comercio de tu barrio registrado en Cauda, comentales así se suman!",
```

Comment:
it should be: "Parece que todavía no hay ningún.."

Reply:
fixed!

Reply:
fixed!